rebench/configurator.py

```diff
@@ -291,9 +307,15 @@ def get_executor(self, executor_name, run_details, variables, action):
             raise ConfigurationError(
                 "An experiment tries to use an undefined executor: %s" % executor_name)
 
+        executor_config = self._executors[executor_name]
         executor = Executor.compile(
-            executor_name, self._executors[executor_name],
+            executor_name, executor_config,
             run_details, variables, self.build_commands, action)
+
+        version_command = executor_config.get('version_command')
+        if version_command:
+            executor.version_command = version_command
+
```

Comment:
This also looks like it should be handled in `model.executor.Executor.compile()`